### DISCUSSION TITLE
change to camel case in lucid

### DISCUSSION THREAD

**Discussion by mahdi2k1** [adonisjs/core#2590] [Help]
@column({ isPrimary: true })
  public id: number;

  @column()
  public email: string;

  @column({ serializeAs: null })
  public password: string;

  @column()
  public fName: string;

  @column()
  public username: string;

  @column()
  public lName: string;

  @column()
  public address: string;

  @column()
  public mobile: string;

  @column()
  public nationalCode: string;

  @column.dateTime({ autoCreate: true })
  public createdAt: DateTime;

  @column.dateTime({ autoCreate: true, autoUpdate: true })
  public updatedAt: DateTime;

-----------------------------------------------

ERROR (mymatno/6120 on DESKTOP-0CC222J): insert into "users" ("address", "created_at", "email", "f_name", "l_name", "mobile", "national_code", "password", "updated_at", "username") values ($1, $2, $3, $4, $5, $6, $7, $8, $9, $10) returning "id" - column "f_name" of relation "users" does not exist

**Comment by RomainLanz** [comment]
Hey @mahdi2k1! 👋 

How do you expect anyone to help you with this amount of information?
You did not even wrote a question.
